src/main/groovy/life/qbic/datamodel/dtos/business/services/ProductUnit.groovy

```diff
@@ -10,7 +10,17 @@ enum ProductUnit {
   PER_GIGABYTE("Gigabyte"),
   PER_SAMPLE("Sample"),
   PER_DATASET("Dataset"),
-  PER_HOUR("Hour")
+  PER_HOUR("Hour"),
+  PER_PROJECT("Project"),
+  PER_RUN("Run"),
+  PER_CYCLE("Cycle"),
+  PER_GEL("Gel/HpH"),
+  PER_MG("10 mg"),
+  PER_MEASUREMENT("Measurement"),
+  PER_CHANNEL("Channel"),
+  PER_PEPTIDE_CHANNEL("100 µg peptides channel"),
+  PER_ML("500 ml"),
```

Comment:
Why is per ml `500 ml` and not `Milliliter` like with Gigabyte?

Reply:
Good Catch!